services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -4937,6 +4937,15 @@ public Map<String, String> getStorageNodesStatus(String clusterName) {
       } else {
         instancesStatusesMap.put(instance, InstanceStatus.DISCONNECTED.toString());
       }
+      Map<String, List<String>> disabledPartitions = helixAdminClient.getDisabledPartitionsMap(clusterName, instance);
```

Comment:
IMHO, I believe it deserves its own command, instead of embedding in a getter function though.